ignite/templates/app/files/cmd/{{binaryNamePrefix}}d/cmd/root.go.plush

```diff
@@ -55,6 +55,7 @@ func NewRootCmd() *cobra.Command {
 	); err != nil {
 		panic(err)
 	}
+	app.AddIBCModuleManager(moduleBasicManager)
```

Comment:
Let's maybe add a comment saying it is necessary until IBC implements depinject properly.